examples/gno.land/r/sys/names/verifier.gno

```diff
@@ -0,0 +1,85 @@
+// Package names provides functionality for checking of package deployments
+// by users registered in r/sys/users are done to proper namespaces.
+package names
+
+import (
+	"std"
+	"strings"
+
+	"gno.land/p/demo/dao"
+	"gno.land/p/demo/ownable"
+
+	"gno.land/r/gov/dao/bridge"
+	"gno.land/r/sys/users"
+)
+
+type verifierFunc func(address std.Address, name string) bool
+
+var (
+	Ownable = ownable.NewWithAddress("g1manfred47kzduec920z88wfr64ylksmdcedlf5") // @moul > dropped in genesis via Enable
```

Comment:
govdao or #3700

Reply:
ohhhhh, no sorry, it's for the enable fuse, that's ok.

Reply:
would be better to be a random address, not mine. ideally one without a private key if this is a genesis msg without signature.